### ISSUE TITLE
Pretrained models or logits of these (Recalibration)

### ISSUE THREAD

**Issue by SebGruber1996** [google/uncertainty-baselines#648]
Hi,

first of all, I really like the idea behind this repository, since it perfectly goes into the right direction of the needs of uncertainty researchers.
One important subfield here is around calibration and recalibration methods.
For a lot of recalibration methods, we do only require the validation and test set logits of the model.
This makes it an annoying nuisance to always have to train a new model from scratch.
Consequently, the following repository is a common source to avoid this https://github.com/markus93/NN_calibration .
Since I see a more promising long term perspective in your project, I would really like to switch, but as long as pretrained weights or the respective logits are not provided, I will have to stick to the other repo for a lot of my work.
Thanks for your efforts!

Greetings,
Sebastian
